README.md

```diff
@@ -1,31 +1,62 @@
 # The OpenAPI Specification
 
-### Draft implementations of tooling supporting the 3.0.0 specification can be found here!
+[![Build Status](https://travis-ci.org/OAI/OpenAPI-Specification.svg?branch=master)](https://travis-ci.org/OAI/OpenAPI-Specification)
 
-[3.0.0 Implementations](https://github.com/OAI/OpenAPI-Specification/blob/OpenAPI.next/IMPLEMENTATIONS.md)
+![](https://avatars3.githubusercontent.com/u/16343502?v=3&s=200)
 
-This is the working branch for the next version of the OpenAPI Specification. You can read more about the Open API Initiative (OAI) at [https://openapis.org](https://openapis.org).
+The OpenAPI Specification is a community driven, open specification within the [Open API Initiative](https://www.openapis.org/), a Linux Foundation Collaborative Project.
 
-The current, released version of the OpenAPI Specification is 2.0, through donation of the Swagger Specification to the OAI by SmartBear Software.  If you are interested in the release specification, see the [master branch](https://github.com/OAI/OpenAPI-Specification/blob/master/README.md) of this project.
+The goal of the OpenAPI Specification is to define a standard, language-agnostic interface description for REST APIs which allows both humans and computers to discover and understand the capabilities of the service without access to source code, documentation, or through network traffic inspection.  When properly defined via OpenAPI, a consumer can understand and interact with the remote service with a minimal amount of implementation logic.  Similar to what interface descriptions have done for lower-level programming, the OpenAPI Specification removes the guesswork in calling the service.
 
-Development of the next version of the OpenAPI Specification is guided by the [OAI Technical Contributors Board](https://github.com/OAI/OpenAPI-Specification/blob/OpenAPI.next/CONTRIBUTORS.md).  This group of committers bring their API expertise, incorporate feedback from the community, and expand the group of committers as appropriate.  All development activity on the future specification will be performed as features and merged into this branch.  Upon release of the OpenAPI Specification, this branch will be merged to master.
+Use cases for machine-readable API interfaces include interactive documentation, code generation for documentation, client, and server, as well as automated test cases. OpenAPI descriptions describe APIs via YAML or JSON documents that adhere to the OpenAPI Specification documented in this repository.  These documents can either be produced and served statically, or be generated dynamically from your application.
```

Comment:
Suggest: "Use cases for machine-readable API interfaces include interactive documentation; code generation for documentation, client, and server; automated test cases."